src/test/java/org/hobbit/core/components/BenchmarkControllerTest.java

```diff
@@ -171,6 +171,7 @@ public void init() throws Exception {
     protected void executeBenchmark() throws Exception {
         // give the start signals
         sendToCmdQueue(Commands.TASK_GENERATOR_START_SIGNAL);
+
```

Comment:
This line is not necessary and it is the only change in this class. So please remove it to reduce the number of unnecessary changes.